src/sentry/issues/occurrence_consumer.py

```diff
@@ -414,6 +416,29 @@ def process_occurrence_group(items: list[Mapping[str, Any]]) -> None:
     Process a group of related occurrences (all part of the same group)
     completely serially.
     """
+
+    if not items:
+        return
+
+    project = Project.objects.get_from_cache(id=items[0]["project_id"])
+    organization = Organization.objects.get_from_cache(id=project.organization_id)
+    if features.has("organizations:occurence-consumer-prune-status-changes", organization):
+        status_changes = [
+            item for item in items if item.get("payload_type") == PayloadType.STATUS_CHANGE.value
+        ]
+
+        if len(status_changes) > 1:
```

Comment:
I think we can just remove the `if`. `status_changes[-1:]` will work even if there are 0 elements in the list. Metric could be `amount=max(0, len(status_changes) - 1)`. Although I'm not sure if a value of 0 is going to do anything other than no-op...

Reply:
I'm leaving this as is, because although `status_changes[-1:]` would work regardless, when theres one or zero status changes, we can avoid recreating `items` list.

Reply:
Right, but don't we still want to put the 1 item at the end of the list?

Reply:
Fair point, I'll update